api/docs/installation/localvir.md

```diff
@@ -13,8 +13,9 @@ $ sudo apt-get update
 Make sure you have the dependencies mentioned above installed before proceeding further.
 For a start, fork BadgeYay to your own github account. Then, clone it to your local system.
 
+* **Step 1**
 ```sh
-$ git clone -b development https://github.com/<your_username>/badgeyay.git
+$ git clone -b development https://github.com//badgeyay.git
```

Comment:
why have you changed this line ? :sweat_smile:

Reply:
Oh That is by mistake , I will resolve it.